client/views/wishCards.ejs

```diff
@@ -6,16 +6,15 @@
 	<!-- ** NOTE: CDN Bootstrap.js, Popper.js, and jQuery are moved to the bottom so the page loads faster-->
 	<%- include('./templates/header')-%>
 	<link rel="stylesheet" href="/public/css/search.css">
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/moment.js/2.29.1/moment.min.js" integrity="sha512-qTXRIMyZIFb8iQcfjXWCO8+M5Tbc38Qi5WzdPOYZHIlZpzBHG3L3by84BBBOiRGiEb7KKtAOAs5qYdUiZiQNNQ==" crossorigin="anonymous"></script>
```

Comment:
please also remove the cdn link to moment :D

Reply:
done